include/ck_tile/ops/fmha/kernel/fmha_bwd_kernel.hpp

```diff
@@ -1047,21 +922,19 @@ struct FmhaBwdDQDKDVKernel
             p_undrop_in_uint8_t = kargs.p_undrop_in_uint8_t;
             drop_seed           = kargs.drop_seed;
             drop_offset         = kargs.drop_offset;
-            is_store_randval    = kargs.is_store_randval;
         }
-        BlockDropout dropout(i_batch,
-                             i_nhead,
-                             kargs.num_head_q,
-                             drop_seed,
-                             drop_offset,
-                             rp_undrop,
-                             p_undrop_in_uint8_t,
-                             is_store_randval);
+        FmhaDropout dropout(i_batch,
+                            i_nhead,
+                            kargs.num_head_q,
+                            drop_seed,
+                            drop_offset,
+                            rp_undrop,
+                            p_undrop_in_uint8_t);
 
```

Comment:
Do we have to construct the FmhaDropout object even when `kHasDropout == false` ?

Reply:
Refactored.